Sources/Turf/Consumer/Consumer+WKT.swift

```diff
@@ -0,0 +1,322 @@
+                .reference(.coordinatesArray)
+            )
+        ])
+    ])
+    
+    /*
+     MULTILINESTRING ((10 10, 20 20, 10 40),
+     (40 40, 30 30, 40 20, 30 10))
+     */
+    private let multiLineString: Consumer<WKTLabel> = .label(.multiLineString, [
+        .spaced(.string("MULTILINESTRING")),
+        .any([
+            .reference(.empty),
+            .bracketed(
+                .reference(.coordinatesArray2D)
+            )
+        ])
+    ])
+    
+    /*
+     POLYGON ((35 10, 45 45, 15 40, 10 20, 35 10),
+     (20 30, 35 35, 30 20, 20 30))
+     */
+    private let polygon: Consumer<WKTLabel> = .label(.polygon, [
+        .spaced(.string("POLYGON")),
+        .any([
+            .reference(.empty),
+            .bracketed(
+                .reference(.coordinatesArray2D)
+            )
+        ])
+    ])
+    
+    /*
+     MULTIPOLYGON (((40 40, 20 45, 45 30, 40 40)),
+     ((20 35, 10 30, 10 10, 30 5, 45 20, 20 35),
+     (30 20, 20 15, 20 25, 30 20)))
+     */
+    private let multiPolygon: Consumer<WKTLabel> = .label(.multiPolygon, [
+        .spaced(.string("MULTIPOLYGON")),
+        .any([
+            .reference(.empty),
+            .bracketed(
+                .reference(.coordinatesArray3D)
+            ),
+        ])
+    ])
+    
+    /*
+     GEOMETRYCOLLECTION (POINT (40 10),
+     LINESTRING (10 10, 20 20, 10 40),
+     POLYGON ((40 40, 20 45, 45 30, 40 40)))
+     */
+    private let geometryCollection: Consumer<WKTLabel> = .label(.geometryCollection, [
+        .spaced(.string("GEOMETRYCOLLECTION")),
+        .any([
+            .reference(.empty),
+            .bracketed(
+                .interleaved(
+                    .spaced(.reference(.object)),
+                    .discard(",")
+                )
+            )
+        ])
+    ])
+    
+    private lazy var object: Consumer<WKTLabel> = {
+        let primitives = empty | number | coordinate | coordinatesArray | coordinatesArray2D | coordinatesArray3D | space
+        let geometries = geometryCollection | point | multiPoint | lineString | multiLineString | polygon | multiPolygon
+        return .label(.object, [
+             primitives | geometries
+        ])
+    } ()
+    
+    private lazy var wktConsumer: Consumer<WKTLabel> = .interleaved([
+        .spaced(object)
+    ], .discard(","))
+    
+    // Transform
+    private let wktTransform: Consumer<WKTLabel>.Transform = { label, values in
+        switch label {
+        case .empty:
+            return nil
+        case .number:
+            guard let numbers = (values as? [String])?.compactMap({ Double($0) }),
+                  numbers.count == values.count,
+                  let number = numbers.first else {
+                throw WKTError.numberParsingFailed(values)
+            }
+            return number
+        case .coordinate:
+            guard let coords = values as? [Double] else {
+                throw WKTError.coordinatesParsingFailed(values)
```

Comment:
Would it be possible to add tests for incorrect WKT input strings? E.g. wrong numbers of elements, incorrect type, etc.

Reply:
Appended